src/App.js

```diff
@@ -1,14 +1,16 @@
 import React, { Component } from 'react';
 import './App.css';
 
-import Jumbotron from './Jumbotron'
+import Jumbotron from './Jumbotron';
+import Form from './Research.js';
```

Comment:
Keep consistant names for your components 
ex: import Reasearch from ./Research (don't specify .js)